addons/udes_stock/models/stock_quant_package.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+
+from odoo import api, fields, models, _
+from collections import defaultdict
+
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class StockQuantPackage(models.Model):
+
+    _name = 'stock.quant.package'
+
+    _inherit = ['stock.quant.package', 'abstract.stock.model']
+
+    CREATE = True
+
+    # This works - but there must be a nicer way
```

Comment:
The default ir.sequence for packages is stock.seq_quant_package, change the prefix using XML.